foxbroadcasting/react-select
# [DPP-817] Clear Text remains after selecting an item for a reference list

## What did you implement:

Implementing issue: [DPP-817](https://fng-jira.fox.com/browse/DPP-817)

## How did you implement it:
When a suggestion is shown, remove it when the user selects it

## How can we verify it:
1. On DPP repo replace the `react-select` line on the `package.json` file with this:
`"@foxdcg/react-select": "git://github.com/foxbroadcasting/react-select.git#bug/DPP-817"`
1. Run `npm i && npm start`
1. Login into the DPP
1. On the navbar select `Series`
1. Select any series
1. On the details sidebar write something in the `Genres` input
1. If you found a word select it and notice the written text is cleared

## Screenshots
![2fcrn2iezx](https://user-images.githubusercontent.com/1103543/47591515-be539980-d924-11e8-9d76-70ab1bf16a6e.gif)


## Checklist
